.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 env:
   JAX_VERSION: 0.4.3
-  TF_VERSION: 2.10.0
+  TF_VERSION: 2.11.0

```

Comment:
I see they have 2.12 out now. Should we consider going all the way to the latest release?

Reply:
Or at least 2.11.1, since it has some security fixes.